packages/core/src/types/database.ts

```diff
@@ -34,8 +34,9 @@ export enum PipelineStatus {
  * The pipeline database parameters list.
  */
 export type PipelineDBParams = 'dataCollectParams' | 'dataAccessParams' | 'dataProcessParams' | 'modelDefineParams' | 'modelLoadParams' | 'modelTrainParams' | 'modelEvaluateParams';
+export type PipelineDBId = 'dataCollectId' | 'dataAccessId' | 'dataProcessId' | 'modelDefineId' | 'modelLoadId' | 'modelTrainId' | 'modelEvaluateId';
```

Comment:
Is this necessary to specify here? The Id should be the only concept inside daemon?

Reply:
Forget to remove it?